sormas-backend/src/main/java/de/symeda/sormas/backend/campaign/CampaignFacadeEjb.java

```diff
@@ -187,6 +197,10 @@ public Campaign fillOrBuildEntity(@NotNull CampaignDto source, Campaign target,
 		return target;
 	}
 
+	@Override
+	protected void delete(Campaign entity) {
```

Comment:
Doesn't this basically break campaign deletion, since there's no code in here?

Reply:
added the campaign delete service to method